### PR TITLE
Updating UserUnblock action to use doExecute().

### PR DESCRIPTION
**Pull request by ndeet**
See d.o. issue https://www.drupal.org/node/2572119

### CODE REVIEW

File: src/Plugin/RulesAction/UserUnblock.php [hunk 1]
```diff
@@ -37,13 +35,14 @@ class UserUnblock extends RulesActionBase {
    */
   protected $saveLater = FALSE;
 
+
```
Comment (klausi) [inline src/Plugin/RulesAction/UserUnblock.php]: unrelated change adding a newline?

**Comment by ndeet** [comment]
Yes sorry, newline now removed :)

**Comment by klausi** [comment]
can you merge in the 8.x-3.x branch?

**Comment by ndeet** [comment]
Done.

**Comment by klausi** [comment]
merged, thanks!
